workers-docs/src/content/reference/apis/kv.md

```diff
@@ -58,11 +58,10 @@ ability to create keys that automatically expire, either at a particular
 point in time or after a certain amount of time has passed since the key was
 last modified.
 
-The core behavior of an expiring key is that once its expiration time has
-been reached, attempts to read it will behave as if the key does not exist.
-Attempting to get the key's value will return a promise that resolves to null
-in a Worker or a 404 HTTP response via the API, and listing keys using the
-API will omit any expired keys from the response.
+Once the expiration time of an expiring key is reached, it will be deleted from
```

Comment:
Presumably there's no billable delete operation involved?

Reply:
Right, they're only billed for delete if they invoke an actual delete call themselves.